examples/rainier/main.bal

```diff
@@ -0,0 +1,386 @@
+    };
+
+    rainier:Employee employee2 = {
+        empNo: "employee-2",
+        firstName: "Jane",
+        lastName: "Doe",
+        birthDate: {year: 1996, month: 9, day: 15},
+        gender: "F",
+        hireDate: {year: 2022, month: 6, day: 1},
+        departmentDeptNo: "department-2",
+        workspaceWorkspaceId: "workspace-2"
+    };
+
+    rainier:Employee employee3 = {
+        empNo: "employee-3",
+        firstName: "Hugh",
+        lastName: "Smith",
+        birthDate: {year: 1986, month: 9, day: 15},
+        gender: "F",
+        hireDate: {year: 2021, month: 6, day: 1},
+        departmentDeptNo: "department-3",
+        workspaceWorkspaceId: "workspace-3"
+    };
+
+    rainier:Employee updatedEmployee1 = {
+        empNo: "employee-1",
+        firstName: "Tom",
+        lastName: "Jones",
+        birthDate: {year: 1994, month: 11, day: 13},
+        gender: "M",
+        hireDate: {year: 2022, month: 8, day: 1},
+        departmentDeptNo: "department-3",
+        workspaceWorkspaceId: "workspace-2"
+    };
+
+    rainier:OrderItem orderItem1 = {
+        orderId: "order-1",
+        itemId: "item-1",
+        quantity: 5,
+        notes: "none"
+    };
+
+    rainier:OrderItem orderItem2 = {
+        orderId: "order-2",
+        itemId: "item-2",
+        quantity: 10,
+        notes: "more"
+    };
+
+    rainier:OrderItem orderItem2Updated = {
+        orderId: "order-2",
+        itemId: "item-2",
+        quantity: 20,
+        notes: "more than more"
+    };
+
+    string[] buildingCodes = check rainierClient->/building.post([building1]);
+    rainier:Building buildingRetrieved = check rainierClient->/building/[building1.buildingCode].get();
+    buildingCodes = check rainierClient->/building.post([building2, building3]);
+    buildingRetrieved = check rainierClient->/building/[building2.buildingCode].get();
+    buildingRetrieved = check rainierClient->/building/[building3.buildingCode].get();
+    buildingRetrieved = check rainierClient->/building/[building1.buildingCode].get();
+    stream<rainier:Building, error?> buildingStream = rainierClient->/building.get();
+    rainier:Building[] buildings = check from rainier:Building building_temp in buildingStream
+        select building_temp;
+    buildingRetrieved = check rainierClient->/building/[building1.buildingCode].put({
+        city: "Galle",
+        state: "Southern Province",
+        postalCode: "10890",
+        'type: "owned"
+    });
+    buildingRetrieved = check rainierClient->/building/[building1.buildingCode].get();
+
+    rainier:Building|error buildingRetrievedError = rainierClient->/building/["invalid-building-code"].put({
+        city: "Galle",
+        state: "Southern Province",
+        postalCode: "10890"
+    });
+    if buildingRetrievedError !is persist:Error {
+        panic error("Error expected");
+    }
+    buildingRetrieved = check rainierClient->/building/[building1.buildingCode].delete();
```

Comment:
```suggestion
    buildingDeleted = check rainierClient->/building/[building1.buildingCode].delete();
```

Reply:
updated in [29c72ce](https://github.com/ballerina-platform/persist-tools/pull/120/commits/29c72ce16d2edf4882433055270167e57360d0f7)